OpenXmlPowerTools/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Looks like you haven't correctly merged your changes with vNext. We removed packages.config and are using new PackageReference in csproj files.

Reply:
yes you are right, I'm fixing it